tests/topotests/lib/common_config.py

```diff
@@ -37,7 +37,7 @@
 import socket
 import ipaddress
 import platform
-
+import ipaddr
```

Comment:
Please do not add ipaddr lib anymore. Use the ipaddress lib which replaced it (and is already imported)